.pre-commit-config.yaml

```
@@ -1,24 +1,36 @@
 repos:
-  - repo: https://github.com/Quantco/pre-commit-mirrors-ruff
-    rev: 0.4.7
+  - repo: local
     hooks:
-      - id: ruff-conda
-        exclude: ^src/glum_benchmarks/orig_sklearn_fork/
-      - id: ruff-format-conda
-        exclude: ^src/glum_benchmarks/orig_sklearn_fork/
-  - repo: https://github.com/Quantco/pre-commit-mirrors-mypy
-    rev: "1.10.0"
-    hooks:
-      - id: mypy-conda
-        args:
-          - --check-untyped-defs
-          - --ignore-missing-imports
-          - --namespace-packages
-        exclude: (^tests/|^src/glum_benchmarks/orig_sklearn_fork/)
-        additional_dependencies: [-c, conda-forge, types-setuptools=67.5, attrs]
-  - repo: https://github.com/Quantco/pre-commit-mirrors-cython-lint
-    rev: 0.16.2
-    hooks:
-      - id: cython-lint-conda
-        args: [--no-pycodestyle]
-      - id: double-quote-cython-strings-conda
+      # ruff
```

```suggestion
      # ensure pixi environments are up to date
      # workaround for https://github.com/prefix-dev/pixi/issues/1482
      - id: pixi-install
        name: pixi-install
        entry: pixi install -e default -e lint
        language: system
        always_run: true
        require_serial: true
        pass_filenames: false
      # ruff
```

please also add this for now until https://github.com/prefix-dev/pixi/issues/1482 is fixed

otherwise you will run into strange errors in pre-commit